jira7/jira_endpoint.bal

```diff
@@ -18,35 +18,32 @@
 
 import ballerina/http;
 
-documentation{ Represents the Jira Client Connector Endpoint configuration.
-    F{{clientConfig}} Http client endpoint configuration
-}
+# Represents the Jira Client Connector Endpoint configuration.
+# + clientConfig - Http client endpoint configuration
 public type JiraConfiguration record {
     http:ClientEndpointConfig clientConfig;
 };
 
-documentation{ Represents the Jira Client Connector Endpoint object.
-    F{{jiraConfig}} jira client Connector endpoint configuration
-    F{{jiraConnector}} jira client connector object
-}
+# Represents the Jira Client Connector Endpoint object.
+# + jiraConfig - jira client Connector endpoint configuration
+# + jiraConnector - jira client connector object
+# + clientConfig - jira client config
```

Comment:
Inconsistent use of cases. If we are referring to the connector in its full name, refer to it as Jira Client Connector. Check all the places